code/modules/projectiles/guns/magic/staff.dm

```diff
@@ -48,11 +48,11 @@
 	recharge_rate = 2
 	no_den_usage = 1
 	school = SCHOOL_FORBIDDEN //this staff is evil. okay? it just is. look at this projectile type list. this is wrong.
-	var/allowed_projectile_types = list(/obj/projectile/magic/change, /obj/projectile/magic/animate, /obj/projectile/magic/resurrection,
+	var/allowed_projectile_types = list(/obj/projectile/magic/animate, /obj/projectile/magic/resurrection,
```

Comment:
SKYRAT EDIT comment?

Reply:
There already is a comment.
Its just at the bottom of the whole list, so I guess it got cut-off.
Check the actual code.